serverlessworkflow/modules/ROOT/pages/job-services/core-concepts.adoc

```diff
@@ -94,7 +79,7 @@ To configure the image by using environment variables you must pass one environm
 .Job Service image configuration for docker execution example
 [source, bash,subs="attributes+"]
 ----
-docker run -it -e JOBS_SERVICE_PERSISTENCE=postgresql -e VARIABLE_NAME=value {jobs_service_image_allinone}:latest
+docker run -it -e QUARKUS_DATASOURCE_USERNAME=postgresql -e VARIABLE_NAME=value {jobs_service_image_postgresql}:latest
```

Comment:
This value "postgresql" differs from the one in the example below. It's probably better to align them
```suggestion
docker run -it -e QUARKUS_DATASOURCE_USERNAME=postgres -e VARIABLE_NAME=value {jobs_service_image_postgresql}:latest
```